src/mlpack/methods/ann/activation_functions/isru_function.hpp

```diff
@@ -0,0 +1,135 @@
+/**
+ * @file isru_function.hpp
+ * @author Prince Gupta
+ *
+ * Definition and implementation of the ISRU function
```

Comment:
Maybe we should mention: ISRU (Inverse Square Root Unit), at least I didn't know what ISRU stands for.